### PR TITLE
Add support for incremental update of PR

### PR DESCRIPTION
**Pull request by smacker**
Part of #259

- update sdk
- update fixtures
- migration with new column and data migration (it's not safe to have them in the same PR if we would have production but OK for staging. Redeployment process is fast and the chance of new PR during it is very low)
- change logic to use new column
- update unit and integration tests

After merge and deployment, we will need 2 more PR to delete `OldInternalID` (unused now). Just delete it from code, re-run kallax code generation, merge. And then create new migration and merge.
Though there is no hurry with it.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -74,7 +74,9 @@
 
 [[constraint]]
   name = "gopkg.in/src-d/lookout-sdk.v0"
-  version = "0.0.4"
+  # there is no release of sdk with new ID for review events yet
+  # version = "0.0.4"
+  revision = "e06a088af4fe40561700c057c5dacd46e23b2abe"
```
Comment (bzz) [inline Gopkg.toml]: @carlosms @dpordomingo would it make sense to release SDK first, as we discussed couple of days ago, and then put 0.0.5 here?

Comment (carlosms) [reply Gopkg.toml]: makes perfect sense, yes.

Comment (bzz) [reply Gopkg.toml]: https://github.com/src-d/lookout-sdk/releases/tag/v0.1.0 🚀

**Comment by smacker** [comment]
sdk version updated. I'm merging after CI is green.

**Pull request merged**